src/index.js

```diff
@@ -0,0 +1,5 @@
+export { default } from './constraints.js';
```

Comment:
This reexports the constraintForUri util as the default export since that's how it was before, but it's a strange default export. I would deprecate that one and also export it as a named export in a future PR.